cypress/support/commands.ts

```
@@ -538,208 +570,131 @@ Cypress.Commands.add(
-            };
-            if (!categoryId) {
-              throw new Error(
-                'seedTestData(actionItemCategories) did not return categoryId.',
-              );
-            }
-            return { categoryId, name };
-          });
-      });
+          .then((res) => ({
+            categoryId: (res as { categoryId: string }).categoryId,
+            name: resolvedName
```

ESLint

[error] Insert `,`

(prettier/prettier)